src/selftest-ro/data/proc/meminfo

```diff
@@ -1,3 +1,7 @@
 MemTotal:        4096 kB
 MemFree:         2048 kB
+Buffers:          512 kB
+Cached:           512 kB
+SReclaimable:       0 kB
```

Comment:
Perhaps, in the future revisions also test that the more complicated formula acts as expected, by having non-zero readings here too?